src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Class of Duke Exceptions.
```

Comment:
This JavaDocs seems a little trivial to me, maybe make it more substantial in the future?

Reply:
Sounds good, thank you.